stubs/stripe/stripe/api_resources/payment_link.pyi

```diff
@@ -0,0 +1,10 @@
+from stripe.api_resources.abstract import (
+    CreateableAPIResource as CreateableAPIResource,
+    ListableAPIResource as ListableAPIResource,
+    UpdateableAPIResource as UpdateableAPIResource,
+)
+
+class PaymentLink(CreateableAPIResource, ListableAPIResource, UpdateableAPIResource):
+    OBJECT_NAME: str
+
+    def list_line_items(self, idempotency_key=None, **params): ...
```

Comment:
This looks like it might be one of those `stripe` methods that's sometimes an instance method and sometimes a classmethod. Am I correct?

Reply:
Yes it is! Is there a preferred way to represent that? I looked through some of the other resources types and didn't find other special handling of `list_line_items` or similar methods.

Reply:
There's no perfect way of representing these in the stubs, unfortunately, as it's a bit too dynamic for type checkers to understand properly! #9814 is a good example of the hack we've been using for some of the other methods `stripe` has that are like this, though.

(As with #9814, you'll have to add an entry to the stubtest allowlist if you want to use that kind of hack, or you'll get a CI error.)

Reply:
Oh very good to know, thank you! Just updated.